aws/buildspecs/development.yaml

```diff
@@ -10,26 +10,19 @@ phases:
     runtime-versions:
       java: corretto8
     commands:
-      - echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee -a  /etc/apt/sources.list.d/sbt.list
-      - curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add
+      - export CODE_ARTIFACT_AUTH_TOKEN=$(aws codeartifact get-authorization-token --domain digiroad --domain-owner 475079312496 --region eu-west-1 --query authorizationToken --output text)
+      - aws codeartifact login --tool npm --repository digiroad_npm_packages --domain digiroad --domain-owner 475079312496
+      - aws s3 cp s3://sbt-repository/sbt-0.13.18.deb /tmp/sbt-0.13.18.deb
```

Comment:
onko tarvetta luoda /tmp/ kansio etukäteen ?

Reply:
Ei pitäis olla tarvetta, toimi ainakin kun testasin aiemmin